proofs/src/program/http.rs

```diff
@@ -15,8 +15,8 @@
 
 use std::collections::HashMap;
 
+use ff::derive::bitvec::macros::internal::funty::Fundamental;
```

Comment:
what does this do 👀

Reply:
I imported it for `As<_> for usize` macros, but that's not ideal. Removed.